### PR TITLE
Add Lincoln Heritage Trail shield

### LINKED ISSUE #568: Lincoln Heritage Trail
The [Lincoln Heritage Trail](https://en.wikipedia.org/wiki/Lincoln_Heritage_Trail) is a single route in Illinois, Indiana, and Kentucky.

As far as I can tell, this route has not yet been mapped as a relation in OpenStreetMap. It is not part of a federal program such as America’s Byways. We don’t have many [examples](https://wiki.openstreetmap.org/wiki/United_States/Unusual_highway_networks) of non-federal, multistate route networks to go off of. The only other example I can think of is the Great Lakes Circle Tour. There are more examples of long-distance cycling route networks that span multiple states without a federal coordinating agency. [`cycle_network=US`](https://wiki.openstreetmap.org/wiki/Tag:cycle_network%3DUS) is being used for this purpose, but such routes don’t really form a single network. Still, based on this tag, maybe `network=US:LHT` would be reasonable for the Lincoln Heritage Trail.

The Lincoln Heritage Trail has [trailblazer](https://www.flickr.com/photos/dougtone/7724385108) and [reassurance](https://commons.wikimedia.org/wiki/File:End_Lincoln_Heritage_Trail_and_Lincoln_Trail_Homestead_State_Park_signs.jpg) markers similar to what we normally include in this style, so we should add a shield definition for it too. It probably would not be possible to make all the shield’s details legible at the normal shield size, but a copper coin with a bust of Lincoln on it should suffice.

[](https://commons.wikimedia.org/wiki/File:End_Lincoln_Heritage_Trail_and_Lincoln_Trail_Homestead_State_Park_signs.jpg)

### PR DESCRIPTION
**Pull request by 1ec5**
Added a shield for the Lincoln Heritage Trail. The icon is very loosely inspired by street-level photos I’ve come across while mapping this route. I decided against IDOT’s white silhouette in https://github.com/ZeLonewolf/openstreetmap-americana/issues/568#issuecomment-1348047708 because there’s too little contrast to discern the figure as anything but a blob.

I took a lot of artistic _liberty_ with this Young Lincoln penny. I started by loosely tracing the Young Lincoln head [this photo of a route marker](https://commons.wikimedia.org/wiki/File:End_Lincoln_Heritage_Trail_and_Lincoln_Trail_Homestead_State_Park_signs.jpg). (The copyright status of that photo is irrelevant because taking a photo of an artistic work [does not confer copyright](https://en.wikipedia.org/wiki/Bridgeman_Art_Library_v._Corel_Corp.) to the photographer, and in any case, there are plenty more just like it.) In a quest to make the coin look vaguely tangible while preserving the style’s overall flat appearance, I took some inspiration from how [the Yoshi coins in _Super Mario_](https://www.mariowiki.com/Coin#Super_Mario_series) depict an embossed effect. Sadly, GL JS does not support spinning animated GIFs: mapbox/mapbox-gl-native#10595.

[Decatur](https://zelonewolf.github.io/openstreetmap-americana/#map=12/39.82335/-88.96325&language=en)

Fixes #568.

### CODE REVIEW

**Comment by ZeLonewolf** [comment]
Should we add #ddcba4 to the palette for flesh-colored shields? :grinning:

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -3107,6 +3107,13 @@ export function loadShields() {
       ))
   );
 
+  // Multistate auto trails
```
Comment (1ec5) [inline src/js/shield_defs.js:3110]: Placing the multistate auto trails down here means they’ll sort below county and township routes in the legend. I considered putting this section right below the federal shields, but most maps relegate auto trails to the bottom of the legend as miscellany anyways.

File: icons/shield_us_lht.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
...414 2.8892293 1.6856852 C 2.891618 1.687929 2.8939518 1.6898766 2.896464 1.6918864 C 2.8989761 1.6938961 2.9015755 1.6958294 2.9042154 1.6975708 C 2.9068554 1.6993122 2.9097117 1.7007829 2.9124837 1.7022216 C 2.9152556 1.7036604 2.9178436 1.7047371 2.9207519 1.705839 C 2.9236602 1.7069409 2.9270048 1.7082088 2.9300536 1.7089396 C 2.9331025 1.7096704 2.9361617 1.7101644 2.9393554 1.7104899 C 2.9425491 1.7108153 2.945831 1.7106041 2.9491739 1.7104899 C 2.9525168 1.7103757 2.9560129 1.7100444 2.9595092 1.7094563 C 2.9630056 1.7088683 2.9661904 1.7084855 2.9698445 1.7073893 C 2.9771526 1.7051968 2.9860146 1.7021553 2.9961995 1.6986043 C 3.0063844 1.6950533 3.0182055 1.6909121 3.0308227 1.6862019 C 3.056057 1.6767816 3.0853425 1.664956 3.1176391 1.6520955 C 3.1499357 1.639235 3.1851888 1.6254597 3.220992 1.6107544 C 3.2567951 1.596049 3.2932417 1.5807508 3.3289957 1.5657959 C 3.4005036 1.5358861 3.4686713 1.5070335 3.5186482 1.4856974 C 3.5686251 1.4643613 3.6008137 1.4505574 3.6008137 1.4505574 C 3.5512048 1.3265341 3.5842773 1.1611688 3.5512043 1.0702189 C 3.5181316 0.97926851 3.079915 0.78909864 2.8070637 0.7312215 C 2.7559041 0.72036945 2.7078873 0.71552479 2.6628865 0.71571857 z"/>
```
[the remark this answers is not in the document]
Comment (ZeLonewolf) [reply icons/shield_us_lht.svg:5]: If there's a second flesh-colored shield component, we should standardize for real though.

File: icons/shield_us_lht.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg width="20" height="20" viewBox="0 0 5.292 5.292" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
+ <circle style="fill:#fff;fill-opacity:1;stroke:#000;stroke-width:.264583;stroke-dasharray:none;stroke-opacity:1" cx="2.646" cy="2.646" r="2.514"/>
+ <path style="fill:#ddcba4;fill-opacity:1;stroke:none;stroke-width:.264583;stroke-dasharray:none;stroke-opacity:1" d="M 2.6458333,4.7624998 A 2.1166666,2.1166666 0 0 1 0.58622172,3.1339702 2.1166666,2.1166666 0 0 1 1.695875,0.75431094 2.1166666,2.1166666 0 0 1 4.2672939,1.2852662"/>
```
Comment (1ec5) [inline icons/shield_us_lht.svg:3]: > If there's a second flesh-colored shield component, we should standardize for real though.

`#ddcba4` is the California MUTCD’s standard “cream” color, also indicated as “buff” or “tan” in some specifications. It complements the brown in our color palette, as intended by Caltrans, but tan/cream/buff is such a rare color on shields that I’m OK with leaving it out of the palette until another case for it arises. We have other one-off deviations from the palette anyways.

/ref https://github.com/ZeLonewolf/openstreetmap-americana/pull/562#discussion_r991796731

File: icons/shield_us_lht.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (1ec5) [reply icons/shield_us_lht.svg:5]: `#ddcba4` is the California MUTCD’s standard “cream” color, also indicated as “buff” or “tan” in some specifications. It complements the brown in our color palette, as intended by Caltrans, but tan/cream/buff is such a rare color on shields that I’m OK with leaving it out of the palette until another case for it arises. We have other one-off deviations from the palette anyways.

/ref https://github.com/ZeLonewolf/openstreetmap-americana/pull/562#discussion_r991796731

[the remark this answers is not in the document]
Comment (1ec5) [reply icons/shield_us_lht.svg:5]: In this case, it isn’t intended to be a skin color, anyhow. The sign depicts the whole disc in the same color (apart from shading). I guess it’s supposed to be a copper or gold color, but in the field it looks more like a brown or Dijon mustard.

**Review by ZeLonewolf — APPROVED** [review]
Looks good to me. The duplicates will go away once openmaptiles/openmaptiles#1361 gets replicated into planetiler.

image

**Pull request merged**